rotkehlchen/globaldb/schema.py

```diff
@@ -220,6 +236,16 @@
 );
 """
 
+# Similar to the common_assets_details this table is used for custom assets that the user wants to
+# to track. Also we use the asset identifier to relate this table with the assets table allowing
+# a delete on cascade. The notes fields allows for adding relevant information about the asset
+# by the user. The type field is a string field that is filled by the user. This allows to create
+# something like tags so the user can visually see what kind of assets (s)he has. All the
```

Comment:
Hm... I would say that it is not tags since there can be only one type and tags is usually something multiple (at least in my mind)